automation/taskcluster/decision_task.py

```diff
@@ -201,30 +193,39 @@ def android_multiarch():
 
 def android_multiarch_release():
     module_build_tasks = build_gradle_modules_tasks(True)
-    return (
-        linux_build_task("All modules - Publish via bintray")
-        .with_dependencies(*module_build_tasks.values())
-        # Our -unpublished- artifacts were uploaded in build_gradle_modules_tasks(),
-        # however there is not way to just trigger a bintray publish from gradle without
-        # uploading anything, so we do it manually using curl :(
-        # We COULD publish each artifact individually, however that would mean if
-        # a build task fails we end up with a partial release.
-        # Since we manipulate secrets, we also disable bash debug mode.
-        .with_script("""
-            python automation/taskcluster/release/fetch-bintray-api-key.py
-            set +x
-            BINTRAY_USER=$(grep 'bintray.user=' local.properties | cut -d'=' -f2)
-            BINTRAY_APIKEY=$(grep 'bintray.apikey=' local.properties | cut -d'=' -f2)
-            PUBLISH_URL=https://api.bintray.com/content/mozilla-appservices/application-services/org.mozilla.appservices/{}/publish
-            echo "Publishing on $PUBLISH_URL"
-            curl -X POST -u $BINTRAY_USER:$BINTRAY_APIKEY $PUBLISH_URL
-            echo "Success!"
-            set -x
-        """.format(appservices_version()))
-        .with_scopes("secrets:get:project/application-services/publish")
-        .with_features('taskclusterProxy') # So we can fetch the bintray secret.
-        .create()
-    )
+
+    version = appservices_version()
+    worker_type = os.environ['BEETMOVER_WORKER_TYPE']
+    bucket_name = os.environ['BEETMOVER_BUCKET']
+    bucket_public_url = os.environ['BEETMOVER_BUCKET_PUBLIC_URL']
+
+    for module_info in module_definitions():
+        module = module_info['name']
+        build_task = module_build_tasks[module]
+        for artifact_info in module_info['artifacts']:
+            artifact_name = artifact_info['name']
+            artifact = artifact_info['path']
```

Comment:
Might as well make this `artifact_path` for consistency; it's weird to have `artifact_info` and then `artifact` for a sub-piece of it.